addons/breathing/stringtable.xml

```diff
@@ -479,21 +454,7 @@
             <Finnish>1x hemopneumotoraksi</Finnish>
         </Key>
         <Key ID="STR_KAT_Breathing_tensionpneumothorax_display">
-            <English>Needle Decompression</English>
-            <German>Nadel Dekompression</German>
-            <Spanish>Descompresión con aguja</Spanish>
-            <Polish>Igła dekompresyjna</Polish>
-            <Chinese>针减压</Chinese>
-            <Chinesesimp>胸腔穿刺减压</Chinesesimp>
-            <Czech>Dekompresní Jehla</Czech>
-            <Korean>바늘 감압</Korean>
-            <French>Aiguille de décompression</French>
-            <Turkish>İğne Dekompresyonu</Turkish>
-            <Italian>Decompressione con ago</Italian>
-            <Russian>Декомпрессия иглы</Russian>
-            <Japanese>胸腔穿刺減圧</Japanese>
-            <Portuguese>Descompressão com Agulha</Portuguese>
-            <Finnish>Neulan dekompressio</Finnish>
+            <English>Finger Thoracostomy (AAT Kit)</English>
```

Comment:
i feel like we should just make separate action using scalpel for that instead of changing current item action name

Reply:
idk how to do that but i was told the AAT kit because it clears hemos.

Reply:
I guess the Devs need to decide if we wanna make the name medically correct or if we just wanna say drain fluid forever. Id like it to say the technically correct thing. (thoracostomy<chest tubes, or finger thoracostomy<no chest tube) id be happy to provide documentation for clarification.